### PR TITLE
Update rxdart dependency to 0.28.0 in alice package

### PR DESCRIPTION
**Pull request by DDavidPrime**
As per issue #234 , I've taken the time to create a PR for this. Let me know if there's anything else I should address if this is something that will be merged.

### CODE REVIEW

File: packages/alice/lib/ui/calls_list/page/alice_calls_list_page.dart [hunk 1]
```diff
@@ -267,7 +267,9 @@ class _AliceCallsListPageState extends State<AliceCallsListPage>
             ? context.i18n(AliceTranslationKey.saveSuccessView)
             : null,
         secondButtonAction: () =>
-            OperatingSystem.isAndroid ? OpenFilex.open(result.path) : null,
+            OperatingSystem.isAndroid && result.path != null
```
Comment (techouse) [inline packages/alice/lib/ui/calls_list/page/alice_calls_list_page.dart:270]: This shouldn't be part of this PR and is already being addressed in https://github.com/jhomlala/alice/pull/229

**Review by techouse — CHANGES_REQUESTED** [review]

File: packages/alice/lib/ui/calls_list/page/alice_calls_list_page.dart (hunk 1 above)
Comment (DDavidPrime) [reply packages/alice/lib/ui/calls_list/page/alice_calls_list_page.dart]: Updated! Do we know when this will be merged?

Comment (techouse) [reply packages/alice/lib/ui/calls_list/page/alice_calls_list_page.dart]: What do you mean by "this"? 😅 This PR here or #229?

Comment (DDavidPrime) [reply packages/alice/lib/ui/calls_list/page/alice_calls_list_page.dart]: Ideally both, but I meant this specific PR. Sorry for not being specific. 🙃

Comment (techouse) [reply packages/alice/lib/ui/calls_list/page/alice_calls_list_page.dart]: It will probably have to come after #229 as without it the CI won't be able to deploy it.

Obviously, I can't approve my own PR (#229) so we'll have to wait on @jhomlala to approve it.

**Review by jhomlala — APPROVED** [review]
